x-pack/plugins/observability/public/pages/alerts/alerts_status_filter.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonGroup, EuiButtonGroupOptionProps } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { AlertStatus, AlertStatusFilter } from '../../../common/typings';
+
+export interface AlertStatusFilterProps {
+  status: AlertStatus;
+  onChange: (id: string, value: string) => void;
+}
+
+export const allAlerts: AlertStatusFilter = {
+  status: AlertStatus.All,
+  query: '',
+  label: i18n.translate('xpack.observability.alerts.alertStatusFilter.showAll', {
+    defaultMessage: 'Show all',
+  }),
+};
+
+export const activeAlerts: AlertStatusFilter = {
+  status: AlertStatus.Active,
+  query: `kibana.alert.status : "active"`,
+  label: i18n.translate('xpack.observability.alerts.alertStatusFilter.active', {
+    defaultMessage: 'Active',
+  }),
+};
+
+export const recoveredAlerts: AlertStatusFilter = {
+  status: AlertStatus.Recovered,
+  query: `kibana.alert.status : "recovered"`,
```

Comment:
@fkanout It is best practice to not hardcode the alerts-as-data field names. We have a technical_field_names file https://github.com/elastic/kibana/blob/main/packages/kbn-rule-data-utils/src/technical_field_names.ts where we have defined all field names. Same goes for alert values as well https://github.com/elastic/kibana/blob/main/packages/kbn-rule-data-utils/src/alerts_as_data_status.ts. 

You can import the files like this

```
import { ALERT_STATUS_RECOVERED } from '@kbn/rule-data-utils/alerts_as_data_status';
import { ALERT_STATUS } from '@kbn/rule-data-utils/technical_field_names';
```

Reply:
I didn't know that. Updated accordingly